### PR TITLE
weenman changes

### PR DESCRIPTION
**Pull request by ericmustin**
made some changes

tl;dr basically `example.py` applies your math to every single US county now, based on 03/26 stats.

some notes:

- `example.py` can be copy/pasta'd to SIR_Clean_328_CC , meant to reflect changes. `python3 example.py` should give you sample output assuming the relevant dependancies are installed. No clue how this would work in a jupyter notebook i just ran it locally

- `example_output.txt` is what output locally for me, looks like it worked, but idk what your math is supposed to do.

- `us_counties.csv` is basically just grabbing the county info from  `https://github.com/nytimes/covid-19-data` . from 03/26, you'd need to update that file every day to stay relevant, the idea is that it just uses the last available days info to do the plotting stuff

-  `population.json` is essentially some formatting done on us census info grabbed via this http req `https://api.census.gov/data/2019/pep/population?get=DATE_DESC,POP,NAME,STATE&for=county:*`

- The json of `population.json` is `key:value` formatted, with the `key` the fips code (which just appears to be a concatenation of state+county codes), and the `value` being the population of the county per 2019 census estimates. This fips code is what's used to connect the nyt infection data to the census population data.

- I didnt really change anything except that i wrapped your math in a function, then i wrote a small helper function that combines the nyt+census data to create a dictionary of counties+infection+population, then i loop over that dictionary and pass each county entry into your math function, using it to set the variables for population and infection. I just straight up removed the adjustment for the company specific related stats u appear to have in there, and i also added one or two pandas and matplot lib related hacks since i was getting a bunch of annoying warning logs when i ran this locally and google told me to i could add a couple 1-liners to remove them ,so that's what i did. yolo.

- On a personal note, Imk if/when u merge this so i can delete my local copy. frankly u should prob have this behind a private gh if ur gonna have work stuff on here, just my two cents

#AlwaysBeTunneling

### CODE REVIEW

**Comment by cbmeco** [comment]
Dosomethingcrazy999
